### DISCUSSION TITLE
Render 0B (zero-byte) files differently

### DISCUSSION THREAD

**Discussion by JDeeth** [community/community#35736] [Other Feature Feedback, Questions, & Ideas] [answered]
![Screenshot_20221011-090747-008](https://user-images.githubusercontent.com/847887/195035109-e826c7b3-036b-4cde-975e-0acb56c671b2.png)
There's nothing to see in `__init__.py` because it is an empty file.

Can you save me a click, and style it differently - italic or greyed-out perhaps?

**Comment by jfullstackdev** [comment, marked as answer]
if you are into that view, you can have a personal copy of that and delete it

but there is also the web editor that is VSCode look-and-feel, if you want that way

![image](https://user-images.githubusercontent.com/47092464/195131117-a012c6e7-91a6-4bbf-a3e9-00d3474c62c0.png)

once you are at the repo, just click the dot `.` 

or click this link, you should be logged-in 

https://github.dev/ferstdigital/cqrs-python-demo

**Reply by JDeeth** [reply]
Wow, that is a revelation! Not exactly optimised for mobile but I'll be using this primarily on PCs going forward!
